@uportal/esco-content-menu/src/components/ContentGrid.vue

```diff
@@ -192,27 +195,29 @@ export default {
     },
     filteredPortlets() {
       const portlets = this.portlets || this.localPortlets;
-      const filterValue = this.filterValue.toLowerCase();
-      const filterCategory = this.filterCategory.toLowerCase();
 
-      if (filterValue === '' && filterCategory === '') {
-        return portlets;
-      }
-
-      return portlets.filter(
-          ({categories, title, name, description}) =>
-            (filterCategory === '' ||
-            categories.some((cat) =>
-              cat.toLowerCase().includes(filterCategory)
-            )) &&
-          (filterValue === '' ||
-            (categories.some((cat) =>
-              cat.toLowerCase().includes(filterValue)
-            ) ||
-              title.toLowerCase().includes(filterValue) ||
-              name.toLowerCase().includes(filterValue) ||
-              description.toLowerCase().includes(filterValue)))
-      );
+      const categoryFilter =
+        this.filterCategory.trim() === ''
+          ? // no filter applied return everything
+            (portlets) => portlets
+          : // filter by category
+            (portlets) =>
+              matchSorter(portlets, this.filterCategory, {
+                keys: ['categories'],
+                threshold: matchSorter.rankings.EQUAL,
```

Comment:
This raises the threshold so category names don't do partial matching or abbreviation matching.